html/webappapis/dynamic-markup-insertion/opening-the-input-stream/bailout-order.window.js

```diff
@@ -0,0 +1,29 @@
+function openHasTakenEffect(doc) {
+  return !doc.childNodes.length;
+}
+
+async_test(t => {
+  const iframe = document.body.appendChild(document.createElement("iframe"));
+  t.add_cleanup(() => { try { iframe.contentDocument.close(); } catch (e) {} });
```

Comment:
When would close throw? Would doing so be OK per spec? Swallowing an exception always deserves a comment, I think.

Reply:
It would throw in this case with an XML document, since `document.close()` is spec'd to throw. However, I don't think every browser does that.